packages/hoppscotch-app/components/environments/Environment.vue

```diff
@@ -41,6 +41,17 @@
             }
           "
         />
+        <SmartItem
+          v-if="!(environmentIndex === 'Global')"
+          svg="copy"
+          :label="`${$t('action.copy')}`"
```

Comment:
I prefer the button should read "Duplicate" instead of "Copy".
Feel free to add a new string entry to our English translation file, here under `action`: https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-app/locales/en.json

Reply:
Yea I agree. Will do the change. 
What about the other translations?

Reply:
You can omit other translations except English. Other will be updated once every week.